mofacts/client/lib/router.js

```diff
@@ -249,8 +249,8 @@ Router.route('/instructions', {
       console.log('No one logged in - allowing template to handle');
     } else {
       const unit = Session.get('currentTdfUnit');
-      const txt = _.chain(unit).prop('unitinstructions').first().trim().value();
-      const pic = _.chain(unit).prop('picture').first().trim().value();
+      const txt = unit.unitinstructions?.trim();
```

Comment:
I appreciate the use of a convenient function here but optional chaining is part of es2020 and we're trying to target es6 right now as many users may have older browsers.  You might do some research and see if we can bump that, I would say we can safely assume browser versions before 3 years ago are off the table for supporting.

Reply:
It may also be the case that the meteor compilation process polyfills this or otherwise negates it being an issue.  You might take a look at that as well. We may be able to tune the output to ensure support of older browsers.

Reply:
It looks like optional chaining isn't supported in browser versions older than 2020, and its not supported at all in IE. 
Meteor will handle the error without crashing but it still throws an exception. Ill just change the optional chaining to something else.

Reply:
So I know babel is somehow involved in the meteor compilation process which is a tool which has the ability to transpile from one es version to another.  Take a look at whether there's some kind of configuration option in meteor that will allow us to make use of that.  It sure would be nice to have this and null coalescing operators

Reply:
Its supported in babel 7.8 but from what I can see we are on 7.7

Reply:
We should be able to bump that probably.  Might have to bump our meteor version a bit